umbraco/Umbraco-CMS
# Fix for potential race condition in packages search

### Description

Fixes [AB#22253](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/22253)

Noticed that the packages section sometimes triggers a race condition, where a slow search would overrule a newer faster search. This fix ensures that any concurrent requests are being aborted by sending in the correct parameters to the resource handler.

### How to test

1. Go to packages section
2. Launch DevTools and set network throttling to "3G slow"
3. Type a word in the search field, and before the request completes, delete the word and type another
4. See that the first request was properly canceled/aborted in the network tab
